android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -726,7 +769,11 @@ private MediaSource buildMediaSource(Uri uri, String overrideExtension, DrmSessi
         int type = Util.inferContentType(!TextUtils.isEmpty(overrideExtension) ? "." + overrideExtension
                 : uri.getLastPathSegment());
         config.setDisableDisconnectError(this.disableDisconnectError);
-        MediaItem mediaItem = new MediaItem.Builder().setUri(uri).build();
+
+        MediaItem mediaItem = new MediaItem.Builder()
+            .setUri(uri)
+            .setAdsConfiguration(new MediaItem.AdsConfiguration.Builder(adTagUrl).build())
```

Comment:
should you test adTagUrl value before calling setAdsConfiguration ?

Reply:
Yes I should and will!